pylons_sphinx_themes/pylons/static/pylons.css_t

```diff
@@ -192,6 +192,40 @@ p.sidebar-title {
     font-weight: bold;
 }
 
+form.search {
+    height: 50px;
+}
+div.sphinxsidebar #searchbox input[type="submit"] {
+    width: 10%;
+}
+
+div.sphinxsidebar .ethical-text, div.sphinxsidebar .ethical-callout {
```

Comment:
I can completely understand why you'd want to customize the CSS to make it better fit with the theme and I'll try to keep you up to date if we change things. We are mostly optimized for the RTD theme and Alabaster but each theme has its own intricacies. We attempt to *rarely* change the main ad class names (`ethical-sidebar`, `ethical-footer`) as we would need to request updates to the [Acceptable Ads filters](https://acceptableads.com/).

Reply:
If you have a repo where you update these things, I can watch it. Otherwise a mention would be sufficient. Please let me know.